src/components/AvatarLogo/AvatarLogo.component.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { useContext } from 'react';
+import Styled from './AvatarLogo.styled';
+import { AccountContext } from '../../contexts/AccountContextProvider';
+import {storage} from '../../utils/storage';
```

Comment:
I don't think `storage` is the right place to put your state here. You have to use ContextAPI or Redux so that when any state change your component can know it and render UI corectly